rs-matter/src/fabric.rs

```diff
@@ -108,7 +108,7 @@ impl Fabric {
             icac,
             noc,
             ipk,
-            label: label.into(),
+            label: label.try_into().unwrap(),
```

Comment:
would it be worth it to be able to mabye map_err instead of unwraping here? I imagine error is unlikely, however it still seems nice to try to not unwrap